test/performance/search/index_construction_benchmark.cpp

```diff
@@ -0,0 +1,132 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+// written for the purpose of the Bch Softwarepraktikum, early 2019 FU Berlin
+// @author: Clemens Cords <clemenscords@fu-berlin.de>
+
+#include <cstring>
+#include <random>
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/io/stream/debug_stream.hpp>
+#include <seqan3/search/fm_index/all.hpp>
+#include <seqan3/alphabet/all.hpp>
+
+using namespace seqan3;
+
+const std::size_t LENGTH = 50;
```

Comment:
We use east const :grin: :nail_care: 
https://mariusbancila.ro/blog/2018/11/23/join-the-east-const-revolution/